ui.R

```diff
@@ -87,6 +87,73 @@ navbarPage("Power Implications for Reduced Sample Size",
                       )
                     )
            ),
+           
+           # ***************************************************
+           # GSD start
+           # ***************************************************
+           tabPanel("GSD",
+                    sidebarLayout(
+                      sidebarPanel(
+                        selectInput(inputId = "designSelect", label="Type of GSD", 
+                                    choices = c("Pocock" = "P", 
+                                                "O'Brien-Fleming"="OF"), 
+                                    selected="Pocock"),

```

Comment:
I'd suggest OF as the default since it is more commonly used for the primary analysis.

Reply:
Hi Dong,
usually, I would agree with you but in the light of COVID-19, I would suggest a Pocock design as we want to increase the chances to stop as early as possible, i.e. based on the data before the outbreak. We also discussed this in the working group and if I remember this correctly, we all felt that Pocock should be used over OBF.